Universal-Federated-Analytics.js

```diff
@@ -177,45 +200,41 @@ function _updateConfig() {
             case 'subagency':
                 oCONFIG.SUB_AGENCY = _value.toUpperCase();
                 break;
-            case 'maincd':
-                _value = _cleanBooleanParam(_value);
-                if ('true' == _value || 'false' == _value)		 /* only override the default if a valid value is passed */
-                    oCONFIG.USE_MAIN_CUSTOM_DIMENSIONS = _value;
-                break;
-            case 'fedagencydim':
-                _value = _cleanDimensionValue(_value);
-				if (''!=_value)
-					oCONFIG.MAIN_AGENCY_CUSTOM_DIMENSION_SLOT = _value.toLowerCase();
+            case 'sitetopic':
+                oCONFIG.SITE_TOPIC = _value;
                 break;
-            case 'fedsubagencydim':
-                _value = _cleanDimensionValue(_value);
-				if (''!=_value)
-					oCONFIG.MAIN_SUBAGENCY_CUSTOM_DIMENSION_SLOT = _value.toLowerCase();
-                break;
-            case 'fedversiondim':
-                _value = _cleanDimensionValue(_value);
-				if (''!=_value)
-					oCONFIG.MAIN_CODEVERSION_CUSTOM_DIMENSION_SLOT = _value.toLowerCase();
+            case 'siteplatform':
+                oCONFIG.SITE_PLATFORM = _value;
                 break;
             case 'parallelcd':
                 _value = _cleanBooleanParam(_value);
-                if ('true' == _value || 'false' == _value)
+                if (true == _value || false == _value)
```

Comment:
For what this does, it should probably use `===` for stricter equality. 

Right now, if `_value` is `1`, this statement would evaluate to `true`, but if `_value` is `100`, it would evaluate to `false`. I don't think that sort of thing is desired.